lib/ship_fosdick/downloader.rb

```diff
@@ -0,0 +1,21 @@
+module ShipFosdick
+  class Downloader
+    def initialize(bucket)
+      @bucket = bucket
+    end
+
+    def download
+      [].tap do |returned_files|
+        bucket.objects.each do |object|
+          next unless object.key.downcase.include?('shp')
```

Comment:
I feel like this is an awesome idea but... I have no clue if Fosdick can read into buckets, can parse the file when it has a specific policy, etc. Is there anyway to figure out the limitation behind Fosdick's ability to read from the bucket without getting on the phone and having a conversation? Do they have some documentation wrapped up in this function?